### PR TITLE
Alibaba: format the log of the destroyer

### PR DESCRIPTION
**Pull request by bd233**
Format the log of the destroyer and output more resource information

### CODE REVIEW

**Review by staebler — COMMENTED** [review]
/lgtm
/approve

**Comment by staebler** [comment]
Is this still a draft? Or is this ready to merge?

**Comment by staebler** [comment]
@bd233 Is this still a draft? Or is this ready to merge?

**Comment by bd233** [comment]
> @bd233 Is this still a draft? Or is this ready to merge?

 I updated most of the logs

File: pkg/destroy/alibabacloud/alibabacloud.go [hunk 1]
```diff
@@ -180,7 +180,7 @@ func (o *ClusterUninstaller) Run() (*types.ClusterQuota, error) {
 func (o *ClusterUninstaller) destroyCluster() error {
 	stagedFuncs := [][]struct {
 		name    string
-		execute func() error
+		execute func(*logrus.Entry) error
```
Comment (staebler) [inline pkg/destroy/alibabacloud/alibabacloud.go:183]:
```suggestion
		execute func(logrus.FieldLogger) error
```

File: pkg/destroy/alibabacloud/alibabacloud.go [hunk 2]
```diff
@@ -237,14 +237,15 @@ func (o *ClusterUninstaller) destroyCluster() error {
...
 	defer wg.Done()
 
 	err := wait.PollImmediateInfinite(
 		time.Second*10,
 		func() (bool, error) {
-			ferr := f.execute()
+			stageLogger := o.Logger.WithField("stage", f.name)
+			ferr := f.execute(stageLogger)
 			if ferr != nil {
 				o.Logger.Debugf("%s: %v", f.name, ferr)
```
Comment (staebler) [inline pkg/destroy/alibabacloud/alibabacloud.go:250]:
```suggestion
				stageLogger.WithError(err).Debugf("Error executing stage")
```

File: pkg/destroy/alibabacloud/alibabacloud.go [hunk 3]
```diff
@@ -358,10 +360,12 @@ func (o *ClusterUninstaller) deleteResourceGroup() (err error) {
 		},
 	)
 
+	logger.Info("Resource groups deleted")
```
Comment (staebler) [inline pkg/destroy/alibabacloud/alibabacloud.go:363]:
```suggestion
	logger.Info("Resource group deleted")
```

File: pkg/destroy/alibabacloud/alibabacloud.go [hunk 4]
```diff
@@ -422,10 +429,12 @@ func (o *ClusterUninstaller) deleteBucket() (err error) {
 		},
 	)
 
+	logger.Info("OSS bucket Deleted")
```
Comment (staebler) [inline pkg/destroy/alibabacloud/alibabacloud.go:432]:
```suggestion
	logger.Info("OSS bucket deleted")
```

File: pkg/destroy/alibabacloud/alibabacloud.go [hunk 5]
```diff
@@ -509,31 +527,33 @@ func (o *ClusterUninstaller) listSlb(slbIDs []string) (response *slb.DescribeLoa
...
 	request.LoadBalancerId = slbID
 	request.DeleteProtection = "off"
 	_, err = o.slbClient.SetLoadBalancerDeleteProtection(request)
 	return
 }
 
-func (o *ClusterUninstaller) deleteSlb(slbID string) (err error) {
-	o.Logger.Debugf("Start to delete SLB %q", slbID)
+func (o *ClusterUninstaller) deleteSlb(slbID string, logger *logrus.Entry) (err error) {
+	logger.WithField("slbID", slbID).Debug("Deleting")
```
Comment (staebler) [inline pkg/destroy/alibabacloud/alibabacloud.go:549]: The logger passed in already has a field "id" that has the SBL ID.

File: pkg/destroy/alibabacloud/alibabacloud.go [hunk 6]
```diff
@@ -578,6 +599,8 @@ func (o *ClusterUninstaller) deleteVSwitchs() (err error) {
 			return false, nil
 		},
 	)
+
+	logger.Info("VSwitch deleted")
```
Comment (staebler) [inline pkg/destroy/alibabacloud/alibabacloud.go:603]:
```suggestion
	logger.Info("VSwitches deleted")
```

File: pkg/destroy/alibabacloud/alibabacloud.go [hunk 7]
```diff
@@ -834,19 +863,20 @@ func (o *ClusterUninstaller) deleteSecurityGroups() (err error) {
...
 	request := ecs.CreateDeleteSecurityGroupRequest()
 	request.SecurityGroupId = securityGroupID
 	_, err = o.ecsClient.DeleteSecurityGroup(request)
 	return
 }
 
-func (o *ClusterUninstaller) deleteSecurityGroupRules(securityGroupID string) (err error) {
-	o.Logger.Debugf("Start to delete security group %q rules ", securityGroupID)
+func (o *ClusterUninstaller) deleteSecurityGroupRules(securityGroupID string, logger *logrus.Entry) (err error) {
+	logger.WithField("securityGroupID", securityGroupID).Debug("Revoking")
```
Comment (staebler) [inline pkg/destroy/alibabacloud/alibabacloud.go:879]: Other places where the security group ID is added as a field, the key "id" is used. Let's stay consistent. Either change everywhere else to use "securityGroupID" or change this place to use "id".
```suggestion
	logger.WithField("id", securityGroupID).Debug("Revoking")
```

**Review by staebler — COMMENTED** [review]
Good improvements. Thanks.

File: pkg/destroy/alibabacloud/alibabacloud.go (hunk 7 above)
Comment (bd233) [reply pkg/destroy/alibabacloud/alibabacloud.go:879]: Have updated^^

**Review by staebler — APPROVED** [review]
/lgtm
/approve

**Comment by mtulio** [comment]
@bd233 as the 5421 was merged, we need to rebase this PR. ptal? Thanks!

**Pull request merged**